etc/kayobe/environments/ci-multinode/apt.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Disable pulling from pulp repositories, as they currently do not have Ubuntu Jammy packages.
+apt_repositories: "{{ [] if ansible_facts['distribution_release'] == 'jammy' }}"
```

Comment:
Would it make more sense to put this in the base apt.yml so that it works for everyone? Could be implemented instead by setting state=absent for jammy?

Reply:
Previous comment on this: https://github.com/stackhpc/stackhpc-kayobe-config/pull/512#discussion_r1192088974